znai-cli/src/main/java/com/twosigma/znai/cli/ZnaiCliApp.java

```diff
@@ -156,6 +157,7 @@ private void generateDocs() {
                 withFileWithLookupPaths("lookup-paths").
                 withFooterPath(config.getSourceRoot().resolve("footer.md")).
                 withExtensionsDefPath(config.getSourceRoot().resolve("extensions.json")).
+                withGlobalReferencesPath(config.getSourceRoot().resolve("references.csv")).
```

Comment:
Last time, you mentioned that you might support json format as well. Let's document if we have a plan to support json format later.

Reply:
where do you propose we document it?

Reply:
`TODO` at the top of `withGlobalReferencesPath()`.